script/munin-httpd

```diff
@@ -93,9 +93,9 @@ It supports three click zooming.
 
 =over 5
 
-=item B<< --port <portnumber> >>
+=item B<< --listen <hostname:port> >>
 
-Sets the port number to run on. [4948]
+Sets the hostname and port to listen on. [*:4948]
```

Comment:
Perfect. It would be nice to have the same level of support as the Apache `Listen`
https://httpd.apache.org/docs/2.4/bind.html